packages/server/lib/hooks/connection/external-post-connection.ts

```diff
@@ -41,7 +41,7 @@ export async function externalPostConnection(
     const activityLogId = await createActivityLog(log);
 
     const logCtx = await logContextGetter.create(
-        { id: String(activityLogId), operation: { type: 'post-connection-script' }, message: 'Start action' },
+        { id: String(activityLogId), operation: { type: 'auth', action: 'post_connection' }, message: 'Start external post connection' },
```

Comment:
I don't know if "Start external post connection" makes sense. Would rather it say "Start external post connection script".

Reply:
oops yes you are right